tr/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,340 @@
 
-&#10230;
+&#10230; Algoritma - Temel Bileşen Analizi (TBA) yöntemi, verilerin aşağıdaki gibi varyansı en üst düzeye çıkararak veriyi k boyutlarına yansıtan bir boyut azaltma tekniğidir:
 
 <br>
 
 **37. Step 1: Normalize the data to have a mean of 0 and standard deviation of 1.**
 
-&#10230;
+&#10230; Adım 1: Verileri ortalama 0 ve standart sapma 1 olacak şekilde normalleştirin.
 
 <br>
 
 **38. Step 2: Compute Σ=1mm∑i=1x(i)x(i)T∈Rn×n, which is symmetric with real eigenvalues.**
 
-&#10230;
+&#10230; Adım 2: Gerçek özdeğerler ile simetrik olan Σ=1mm∑i=1x(i)x(i)T∈Rn×n hesaplayın.
 
 <br>
 
 **39. Step 3: Compute u1,...,uk∈Rn the k orthogonal principal eigenvectors of Σ, i.e. the orthogonal eigenvectors of the k largest eigenvalues.**
 
-&#10230;
+&#10230; Adım 3: u1, ...,uk∈Rn 'yi hesaplayın, Σ ort'nin ortogonal ana özvektörlerini, yani k en büyük özdeğerlerin ortogonal özvektörlerini.
 
 <br>
 
 **40. Step 4: Project the data on spanR(u1,...,uk).**
 
-&#10230;
+&#10230; Adım 4: spanR (u1, ..., uk) üzerindeki verileri gösterin.
 
 <br>
 
 **41. This procedure maximizes the variance among all k-dimensional spaces.**
 
-&#10230;
+&#10230; Bu yöntem tüm k-boyutlu uzaylar arasındaki varyansı en üst düzeye çıkarır.
 
 <br>
 
 **42. [Data in feature space, Find principal components, Data in principal components space]**
 
-&#10230;
+&#10230; [Öznitelik uzayında veri, Temel bileşenleri bul, Temel bileşenler uzayında veri]
 
 <br>
 
 **43. Independent component analysis**
 
-&#10230;
+&#10230; Bağımsız bileşen analizi
 
 <br>
 
 **44. It is a technique meant to find the underlying generating sources.**
 
-&#10230;
+&#10230; Temel oluşturan kaynakları bulmak için kullanılan bir tekniktir.
 
 <br>
 
 **45. Assumptions ― We assume that our data x has been generated by the n-dimensional source vector s=(s1,...,sn), where si are independent random variables, via a mixing and non-singular matrix A as follows:**
 
-&#10230;
+&#10230; Varsayımlar - Verilerin x'in n boyutlu kaynak vektörü s=(s1, ..., sn) tarafından üretildiğini varsayıyoruz, burada si bağımsız rasgele değişkenler, bir karışım ve tekil olmayan bir matris A ile aşağıdaki gibi:
 
 <br>
 
 **46. The goal is to find the unmixing matrix W=A−1.**
 
-&#10230;
+&#10230; Amaç, işlem görmemiş matrisini W=A−1 bulmaktır.
 
 <br>
 
 **47. Bell and Sejnowski ICA algorithm ― This algorithm finds the unmixing matrix W by following the steps below:**
 
-&#10230;
+&#10230; Bell ve Sejnowski ICA algoritması - Bu algoritma, aşağıdaki adımları izleyerek işlem görmemiş matrisi W'yi bulur:
 
 <br>
 
 **48. Write the probability of x=As=W−1s as:**
 
-&#10230;
+&#10230; X=As=W−1s olasılığını aşağıdaki gibi yazınız:
 
 <br>
 
 **49. Write the log likelihood given our training data {x(i),i∈[[1,m]]} and by noting g the sigmoid function as:**
 
-&#10230;
+&#10230; Eğitim verisi {x(i),i∈[[1, m]]} ve sigmoid fonksiyonunu g olarak not ederek log olasılığını yazınız:
```

Comment:
Eğitim verisi {x(i),i∈[[1, m]]} ve g sigmoid fonksiyonunu not ederek log olasılığını yazınız: